website/docs/learn/1-setting-up.md

```diff
@@ -12,7 +12,7 @@ If you are **new to dbt**, we recommend you take some time to read through the F
 If you are already **experienced with dbt**, you still need to complete this tutorial as we'll be building on this project in our lessons.
 
 :::info
-Please note that this tutorial is adapted from the main <a href="https://tutorial.getdbt.com/tutorial/setting-up/">Getting Started</a> tutorial — there may be some references to BigQuery in various videos, however the same concepts should apply on Snowflake.
+Please note that this tutorial is adapted from the main <a href="https://tutorial.getdbt.com/tutorial/getting-started">Getting Started</a> tutorial — there may be some references to BigQuery in various videos, however the same concepts should apply on Snowflake.
```

Comment:
Should we be updating this to say concepts apply across data platforms? To be more inclusive of data warehouses outside of BQ and Snowflake